xtokens/src/lib.rs

```diff
@@ -394,6 +394,53 @@ pub mod module {
 
 			Self::do_transfer_multiassets(who, assets.clone(), fee.clone(), dest, dest_weight, true)
 		}
+
+		/// Transfer currency and use relay-chain asset as fee
```

Comment:
Need more descriptions about which edge case this call handles. Refer to docstrings in other dispatchable calls.

Reply:
changes in `do_transfer_multiassets `